src/contribute/release-guide.md

```diff
@@ -289,30 +355,34 @@ Copy the source release to the dev repository of `dist.apache.org`.
 
 1. Make a directory for the new release:
 
-        mkdir beam/${VERSION}
-        cd beam/${VERSION}
+        mkdir beam/${RELEASE}
+        cd beam/${RELEASE}
 
-1. Copy and rename the Beam source distribution, hashes, and GPG signature:
+1. Download source zip from GitHub:
 
-        cp ${BEAM_ROOT}/target/apache-beam-${VERSION}-source-release.zip .
-        cp ${BEAM_ROOT}/target/apache-beam-${VERSION}-source-release.zip.asc .
-        cp ${BEAM_ROOT}/sdks/python/target/apache-beam-${VERSION}.zip apache-beam-${VERSION}-python.zip
+    wget https://github.com/apache/beam/archive/release-${RELEASE}.zip \
+         -O apache-beam-${RELEASE}-source-release.zip
 
-1. Create hashes for source files and sign the python source file file
+1. Create hashes and sign the source distribution:
 
-        sha512sum apache-beam-${VERSION}-source-release.zip > apache-beam-${VERSION}-source-release.zip.sha512
-        gpg --armor --detach-sig apache-beam-${VERSION}-python.zip
-        sha512sum apache-beam-${VERSION}-python.zip > apache-beam-${VERSION}-python.zip.sha512
+        gpg --armor --detach-sig apache-beam-${RELEASE}-source-release.zip
+        sha512sum apache-beam-${RELEASE}-source-release.zip > apache-beam-${RELEASE}-source-release.zip.sha512
 
 1. Add and commit all the files.
 
-        svn add beam/${VERSION}
+        svn add beam/${RELEASE}
         svn commit
 
 1. Verify that files are [present](https://dist.apache.org/repos/dist/dev/beam).
 
```

Comment:
Need a step to build the python binaries and stage on dist.apache.org/dev/beam, like https://dist.apache.org/repos/dist/dev/beam/2.5.0/